content/en/docs/concepts/architecture/mixed-version-proxy.md

```diff
@@ -0,0 +1,84 @@
+---
+reviewers:
+- jpbetz
+title: Mixed Version Proxy
+content_type: concept
+weight: 220
+---
+
+<!-- overview -->
+{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}
+
+Kubernetes {{< skew currentVersion >}} includes an alpha feature that lets one
+{{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
+proxy a resource request to a different _peer_ API server. This is useful when there are multiple
```

Comment:
“a API server”? Usually “an API server” (I prefer “a kube-apiserver”, because this is a feature of _our_ implementation and not a generic aspect of the Kubernetes API).